README.md

```diff
@@ -1 +1,76 @@
-# breaks ur prod
+# Pipelines
+
+## LinkedIn Pipeline and Profile Management
+
+This project is a web application that provides functionality to manage pipelines and profiles on LinkedIn. It leverages MongoDB as the database, Nodejs and Expressjs for the backend, and Reactjs for the frontend.
+
+## Features
+
+1. **Get Pipelines**: Retrieve a list of pipelines.
+
+2. **Get Pipelines By Company**: Fetch pipelines associated with a specific company.
+
+3. **Get Random Pipelines**: Get a random set of pipelines.
+
+4. **Remove Experience on LinkedIn**: Remove existing experience entries on LinkedIn profiles.
+
+5. **Add LinkedIn Experience**: Add new experience entries to LinkedIn profiles.
```

Comment:
I'm not sure what this means but I would reword it so that its clear that our changes only effect our site